test/apiv2/20-containers.at

```diff
@@ -545,6 +545,16 @@ if ! grep -q '201 Created' "${TMPD}/headers.txt"; then
     exit 1
 fi
 
+if root; then
+  podman run -dt --name=updateCtr alpine
+  t POST libpod/containers/updateCtr/update \
+  Memory.Limit=1000 \
+  CPU.Shares=123 \
+  201 \
```

Comment:
Undesired trailing backslash.

And, could you add a test to confirm that the settings took effect?

Reply:
I tried for a while with this. The issue is, there is no exec endpoint and running, podman exec in these APIv2 tests fails claiming the container is not running but executing a podman start erases the update since it is not persistent.

Reply:
fixed this